packages/web-api/src/types/request/chat.ts

```diff
@@ -184,13 +187,19 @@ export type ChatPostMessageArguments = TokenOverridable &
   Unfurls & {
     /** @description Disable Slack markup parsing by setting to `false`. Enabled by default. */
     mrkdwn?: boolean;
+    /** @description Accepts message text formatted in markdown. This argument should not be used
+     * in conjunction with blocks or text. Limit this field to 12,000 characters. */
+    markdown_text?: string;
   };
 
 // https://docs.slack.dev/reference/methods/chat.scheduleMessage
 export type ChatScheduleMessageArguments = TokenOverridable &
   MessageContents & {
     /** @description Unix EPOCH timestamp of time in future to send the message. */
     post_at: string | number;
+    /** @description Accepts message text formatted in markdown. This argument should not be used
+     * in conjunction with blocks or text. Limit this field to 12,000 characters. */
+    markdown_text?: string;
```

Comment:
suggestion: Including a `jsdoc` **example** might be nice too? From the [docs](https://docs.slack.dev/reference/methods/chat.scheduleMessage/) I found:

```
Example: **This is bold text**
```